lib/ex_doc/formatter/html/autolink.ex

```diff
@@ -399,8 +399,9 @@ defmodule ExDoc.Formatter.HTML.Autolink do
     modules_refs = options[:modules_refs] || []
 
     fn all, text, match ->
-      pmfa = {_, module, _, _} = split_match(:module, match)
+      pmfa = {_, original_module, _, _} = split_match(:module, match)
```

Comment:
I don’t understand why we removed the stripping from the split_match functions, since we don’t use original_module.

Reply:
We use original_module to build the text. That's the whole point of not stripping it in split_match but outside of it.

Reply:
Ah, sorry. I somehow missed it. :D